.github/workflows/ci.yml

```diff
@@ -30,7 +30,7 @@ jobs:
 
       - name: Run Nuke
         env:
-          OCTOVERSION_CurrentBranch: ${{ github.head_ref || github.ref_name }}
+          OCTOVERSION_CurrentBranch: ${{ github.event_name == 'schedule' &&  'scheduledRun' || ( github.head_ref || github.ref_name) }}
```

Comment:
Consider doing something like [this](https://github.com/OctopusDeploy/TinyTypes/blob/71625dbe4046108eef3b7775deac159a9ad8825a/.github/workflows/main.yaml#L44).

(value in consistency etc)

Reply:
BFT has some docs around how they do it [here](https://octopushq.atlassian.net/wiki/spaces/RND/pages/2856517684/Sensible+Defaults+for+Builds+in+PET)